chaste_codegen/data/tests/all_models_reference_models/BE/FaberRudy2000.cpp

```diff
@@ -1,6 +1,6 @@
 //! @file
 //!
-//! This source file was generated from CellML by chaste_codegen version 0.4.0
+//! This source file was generated from CellML by chaste_codegen version 0.4.1
```

Comment:
We've discussed before hiding changes from commits where the only difference is a version number like this. Worth considering here?

Reply:
I did apply your suggestions back than but it doesn't seem to be working, although it does seem to work for the date...

Reply:
aah found it!